.github/workflows/release.yml

```diff
@@ -239,7 +239,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ needs.prepare_release.outputs.upload_url }}
-          asset_path: ${{env.GRADLE_DIR}}/${{ env.MAIN_PROJECT_MODULE }}/build/outputs/apk/release/PhoneVR.apk
+          asset_path: ${{env.GRADLE_DIR}}/${{ env.MAIN_PROJECT_MODULE }}/build/outputs/apk/gvr/release/PhoneVR.apk
```

Comment:
Did you miss releasing PhoneVR-noGVR.apk here ? release.yml is responsible for making github-releases, so we need to build no-gvr.apk and upload it into github-releases or did i miss something ?

Reply:
I did miss it

Reply:
Done